src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,57 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+type Props = {
+  searchInput: string;
+  setSearchInput: (value: string) => void;
+  selectOption: string;
+  setSelectOption: (option: string) => void;
+};
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+export const TodoFilter: React.FC<Props> = ({
+  searchInput,
+  setSearchInput,
+  selectOption,
+  setSelectOption,
+}) => {
+  return (
+    <form className="field has-addons">
+      <p className="control">
+        <span className="select">
+          <select
+            data-cy="statusSelect"
+            value={selectOption}
+            onChange={e => setSelectOption(e.target.value)}
```

Comment:
Consider adding a label or aria-label to the select element for better accessibility. This will help screen readers to identify the purpose of the dropdown.